client/client.go

```diff
@@ -544,7 +544,19 @@ func (o *OpResult) String() string {
 	}
 
 	if v := o.OperationID; v != 0 {
-		buf.WriteString(fmt.Sprintf(" AFTOperation { ID: %d, Type: %s, Status: %s }", v, o.Details.Type, o.ProgrammingResult))
+    var key string
```

Comment:
Indent?

Reply:
`gofmt` run - not sure why it didn't before!